.github/workflows/quic.yml

```diff
@@ -117,40 +116,82 @@ jobs:
       ref: ${{ github.event.client_payload.sha || github.event.client_payload.ref || inputs.ref || 'main' }}
 
   #
-  # Test Jobs
+  # Manage Azure VMs, TODO: blocks if resources are not available. Waits for it to become available.
+  #
+  manage-azure-vms:
+    name: Manage Azure VMs
+    runs-on: windows-latest
+    outputs:
+      full-matrix: ${{ steps.generate-full-matrix.outputs.matrix }}
+      azure-matrix: ${{ steps.generate-azure-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install Latest Az PowerShell Module
+        shell: pwsh
+        run: |
+          Install-Module -Name Az -Force -AllowClobber -Repository PSGallery
+          Update-Module Az -Force
+          Get-Module 'Az' | where {([string]($_.Version)).StartsWith('9.3.0')} | Remove-Module
+          Get-Module -Name Az -ListAvailable
+      - name: Login to Azure
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+          enable-AzPSSession: true
+      - name: Run manage-azure-vms.ps1
+        run: ./.github/workflows/manage-azure-vms.ps1 -GithubPatToken ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        shell: pwsh
+      - id: generate-full-matrix
+        run: echo "::set-output name=matrix::$(jq -c '.' ./.github/workflows/processed-matrix.json)"
```

Comment:
I see this warning:
```
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

Reply:
Fixed with [b074ffd](https://github.com/microsoft/netperf/pull/156/commits/b074ffdf19a44f0bc808e16d59ff855c5ebd5ba0)